kbase-extension/static/kbase/js/widgets/narrative_core/jobCommChannel.js

```diff
@@ -194,7 +191,15 @@ define([
                 resolve();
             }).catch((err) => {
                 console.error('ERROR sending comm message', err, msgType, jobId, options);
-                throw new Error('ERROR sending comm message', err, msgType, jobId, options);
+                throw new Error(
+                    'ERROR sending comm message: ' +
+                        JSON.stringify({
+                            error: err,
+                            msgType: msgType,
+                            jobId: jobId,
+                            options: options,
+                        })
```

Comment:
incorrect Error() syntax -- turn into a data structure that can be printed out and read